solver/src/solver/http_solver.rs

```diff
@@ -52,6 +52,7 @@ lazy_static! {
 #[derive(Debug, Default)]
 pub struct SolverConfig {
     pub max_nr_exec_orders: u32,
+    pub uniform_clearing_prices_enforced_via_flag: bool,
```

Comment:
I feel like this field name could be shortened to something like `enforce_uniform_clearing_prices`. Particularly, the that that it is a flag is inherent from the type being boolean.

Reply:
Or use the same name from the query.

Reply:
Agreed. I would either call it:
```suggestion
    pub enforce_uniform_clearing_prices: bool,
```

Or as @vkgnosis suggested:
```rust
pub struct SolverConfig {
    pub max_nr_exec_orders: u32,
    pub uniform_clearing_price_policy: UcpPolicy,
}

pub enum UcpPolicy {
    OrdersOnly,
    Enforce,
}
```

Reply:
thanks for the feedback. I changed it.

The reason I wasn't going for this is at the beginning was that after this change am calling:
```
SolverType::Mip => shared(create_http_solver(
                    account,
                    mip_solver_url.clone(),
                    "Mip",
                    SolverConfig {
                        max_nr_exec_orders: 100,
                        enforce_uniform_clearing_prices: false,
                    },
                ))
                ```
                and then it reads like the uniform clearing prices would not be enforced on the MIP solver, but they are the default for the mip solver and hence they are enforced. That is the reason I was stating : via_flag